lib/dispatcher.rb

```diff
@@ -29,15 +29,15 @@ def email_sent_confirmation(proposal)
 
   def deliver_new_proposal_emails(proposal)
     proposal.currently_awaiting_approvals.each do |approval|
-      self.email_approver(approval)
+      email_approver(approval)
     end
-    self.email_observers(proposal)
-    self.email_sent_confirmation(proposal)
+
+    email_observers(proposal)
+    email_sent_confirmation(proposal)
   end
 
   def deliver_attachment_emails(proposal)
     proposal.users.each do |user|
-      # do not send email to approvers who have not yet heard about the proposal
```

Comment:
Why remove this comment (and the one below)? It seems useful, short of doing https://www.pivotaltracker.com/story/show/104647900.